src/lib/fees/model.ts

```diff
@@ -64,7 +69,7 @@ export const calculateFees: CalculateFeeBps = ({ tradeAmountUsd, foxHeld, feeMod
     new Date().getUTCFullYear() < THORSWAP_MAXIMUM_YEAR_TRESHOLD
 
   // failure to fetch fox discount results in free trades.
-  const isFallbackFees = selectIsSnapshotApiQueriesRejected(store.getState())
+  const isFallbackFees = isSnapshotApiQueriesRejected
```

Comment:
There were circular dependencies because this file was importing redux, but redux slice was import this file ♻️. To break the cycle I opted to just add a parameter